spec/views/posts/index_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe 'User post index page', type: :feature do
+  let!(:user) { User.create(name: 'salma', bio: 'I am a software engineer', photo: 'https://images.unsplash.com/photo-1612833603922-3b3b3b4b4b4b?ixid=MnwxMjA3fDB8MHxzZWFyY2h8Mnx8c29mdHdhcmUlMjBlbmdpbmVlcmluZ3xlbnwwfHwwfHw%3D&ixlib=rb-1.2.1&w=1000&q=80') }
+  let!(:post) { Post.create(title: 'My first post', text: 'This is my first post', author: user) }
+  let!(:comment) { Comment.create(text: 'This is my first comment', author: user, post:) }
+  let!(:like) { Like.create(author: user, post:) }
+
+  before do
+    visit user_posts_path(user)
+  end
+
+  it 'displays the user name' do
+    expect(page).to have_content(user.name)
+  end
+
+  it 'displays the user photo' do
+    expect(page).to have_css("img[src*='#{user.photo}']")
+  end
+
+  it 'displays the number of posts the user has' do
+    expect(page).to have_content("Number of posts: #{user.posts.count}")
+  end
+
+  it 'displays the posts title' do
+    expect(page).to have_content(post.title)
+  end
+
+  it 'displays some of the posts body' do
+    expect(page).to have_content(post.text)
+  end
+
+  it 'displays the first comment on the post' do
+    expect(page).to have_content(comment.text)
+  end
+
+  it 'displays the number of comments the post has' do
+    expect(page).to have_content("comments: #{post.comments_counter}")
+  end
```

Comment:
- [ ] You have done a very good job testing your app but you missed just one which is `Tests that asserts you can see a section for pagination if there are more posts than fit on the view`. kindly add it